plugins/poll/assets/stylesheets/common/poll.scss

```diff
@@ -119,6 +121,37 @@ div.poll {
       margin-right: 0.25em;
     }
   }
+
+  .poll-results-chart {
+    height: 300px;
+    overflow: auto;
+  }
+
+  .poll-group-by-toggle {
+    float: left;
```

Comment:
would really prefer we try to code with flexbox and not floats

Reply:
Bumping this, @jjaffeux is right here.